conformance/packages/dns-test/src/templates/pdns.forwarder.conf.jinja

```diff
@@ -0,0 +1,38 @@
+recursor:
+  write_pid: false
+  socket_dir: /var/run/pdns-recursor
+  # Disable security update polling
+  security_poll_suffix: ""
+
+  forward_zones_recurse:
```

Comment:
note: we want `forward_zones_recurse` here, not `forward_zones` since we're forwarding to a recursor, and not an auth server. Otherwise the RD bit isn't set on the forwarded query and the resolver won't process it.